packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -0,0 +1,204 @@
+import os
+from collections.abc import Collection, Iterable, Sequence
+from typing import TYPE_CHECKING, Any, Callable, NamedTuple, Optional
+
+import juliacall
+from juliacall import Main as jl
+from typing_extensions import TypeAlias
+
+from quri_parts.core.estimator import (
+    ConcurrentParametricQuantumEstimator,
+    ConcurrentQuantumEstimator,
+    Estimatable,
+    Estimate,
+    ParametricQuantumEstimator,
+    QuantumEstimator,
+    create_parametric_estimator,
+)
+from quri_parts.core.operator import zero
+from quri_parts.core.state import CircuitQuantumState, ParametricCircuitQuantumState
+from quri_parts.core.utils.concurrent import execute_concurrently
+
+from .circuit import convert_circuit
+from .operator import convert_operator
+
+if TYPE_CHECKING:
+    from concurrent.futures import Executor
+
+abs_dir = os.path.dirname(os.path.abspath(__file__))
+library_path = os.path.join(abs_dir, "library.jl")
+include_statement = 'include("' + library_path + '")'
+jl.seval(include_statement)
+
+
+class _Estimate(NamedTuple):
+    value: complex
+    error: float = 0.0
```

Comment:
I'm not sure if ITensor is an exact simulator. Usually TN calculation involves approximation when creating MPS. If that is the case in ITensor as well , `error=0.0` might cause misunderstanding.
You can set `None` if it's not exact.